src/components/articles/ArticleNotFound.jsx

```
@@ -0,0 +1,76 @@
+
+    const alertMessage = `Component <strong>${dataWrapper.component}</strong> not found! Make sure the component exists and is listed on the <i>SectionBody.ARTICLES</i> dictionary on <b>SectionBody.jsx</b>.`
+
+    return (
+        <Article id={dataWrapper.uniqueId}
+                 type={Article.Types.SPACING_DEFAULT}
+                 dataWrapper={dataWrapper}
+                 className={`article-not-found`}
+                 selectedItemCategoryId={selectedItemCategoryId}
+                 setSelectedItemCategoryId={setSelectedItemCategoryId}>
+            <Alert variant={`danger`} className={`text-3 mb-0`}>
+                <span dangerouslySetInnerHTML={{__html: alertMessage}}/>
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)